docs/IR.md

```diff
@@ -38,7 +38,11 @@ Further metadata may be added to a model via its `metadata_props` field (as desc
 
 ### Graphs
 
-Each computation dataflow graph is structured as a list of nodes that form a graph. These nodes MUST be free of cycles. Each node is a call to an operator. Nodes have one or more inputs, one or more outputs, and zero or more attribute-value pairs.
+Each computation dataflow graph is structured as a list of nodes that form a graph. Each node is a call to an operator. Nodes have one or more inputs, one or more outputs, and zero or more attribute-value pairs. Additionally, a valid ONNX graph must fullfil the following requirements:
```

Comment:
Rereading this... don't constant nodes have no inputs? :)

Reply:
Yep good catch